mediaflow_proxy/extractors/lulustream.py

```
@@ -0,0 +1,27 @@
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.mediaflow_endpoint = "hls_manifest_proxy"
+
+    async def extract(self, url: str, **kwargs) -> Dict[str, Any]:
+        response = await self._make_request(url)
+
+        # See https://github.com/Gujal00/ResolveURL/blob/master/script.module.resolveurl/lib/resolveurl/plugins/lulustream.py
+        pattern = r'''sources:\s*\[{file:\s*["'](?P<url>[^"']+)'''
+        match = re.search(pattern, response.text, re.DOTALL)
+        if not match:
+            raise ExtractorError("Failed to extract source URL")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)